tcblib/db_functions.py

```diff
@@ -0,0 +1,97 @@
+import boto3
+import datetime
+import os
+
+DB = boto3.client('dynamodb')
+TABLE = os.environ['DYNAMODB_TABLE']
+DEFAULT_HOST_ID = os.environ['DEFAULT_HOST_ID']
+DEFAULT_HOST_NAME = os.environ['DEFAULT_HOST_NAME']
+
+def get_date():
+    """Get date for next tech chat (will return today if today is Friday)."""
+    today = datetime.date.today()
+    friday = today + datetime.timedelta((4-today.weekday()) % 7)
+    return friday.strftime('%Y%m%d')
+
+
+def query_host(user_attr):
+    """Query for the host, for either the name or user_id."""
+    response = DB.get_item(
+        TableName=TABLE,
+        Key={
+            'Date': {'N': '{0}'.format(get_date())}
+        },
+        ProjectionExpression='Host.#attr',
+        ExpressionAttributeNames={
+            '#attr': user_attr
+        }
+    )
```

Comment:
Y'know I made this so it could be used to get the ID for the DM but it never even gets used so maybe this is a lil over-engineered/redundant.

Reply:
when you say it, do you mean the method in general? You do use it! If you dont use it, i'd def default to YAGNI.  Because its in version control, you can always peak at the code later.

Reply:
o i use the method, but never use it to get the host ID [which is why it was written in a way so you could get either the human-readable hostname _or_ the id...but then it turned out i never called this method to get the id, the human-readable name]

Reply:
Then I'd nix the part that allows you to get the ID.